SIMS/SIMS/obj/Debug/net8.0/SIMS.AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Reflection;
+
+[assembly: System.Reflection.AssemblyCompanyAttribute("SIMS")]
+[assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
+[assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
+[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0+119205794909a7b4478160b56387fa7fe664872e")]
+[assembly: System.Reflection.AssemblyProductAttribute("SIMS")]
+[assembly: System.Reflection.AssemblyTitleAttribute("SIMS")]
+[assembly: System.Reflection.AssemblyVersionAttribute("1.0.0.0")]
+
+// Generated by the MSBuild WriteCodeFragment class.
+
```

Comment:
we do not need to track this file ( debug files) let's add them to .gitignore file